packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -515,5 +520,92 @@ describe('masternode', () => {
       expect(typeof data).toBe('string')
       expect(data.length).toBe(64)
     })
+
+    it('should not utxosToAccount with utxos for DFI coin if does not own the recipient address', async () => {
+      const payload: UtxosToAccountPayload = {}
+      // NOTE(jingyi2811): Only support sending utxos to DFI account.
+      payload['2Mywjs9zEU4NtLknXQJZgozaxMvPn2Bb3qz'] = '5@DFI'
+
+      const utxos = await container.call('listunspent')
+      const inputs = utxos.map((utxo: { txid: string, vout: number }) => {
+        return {
+          txid: utxo.txid,
+          vout: utxo.vout
+        }
+      })
+
+      const promise = client.account.utxosToAccount(payload, inputs)
+
+      await expect(promise).rejects.toThrow(RpcApiError)
+      await expect(promise).rejects.toThrow('The address (2Mywjs9zEU4NtLknXQJZgozaxMvPn2Bb3qz) is not your own address')
+    })
+  })
+
+  describe('accountToAccount', () => {
+    it('should accountToAccount', async () => {
+      const from = await container.getNewAddress()
+      await createToken(from, 'DABC', 5)
+      await createToken(from, 'DDEF', 5)
+
+      const payload: UtxosToAccountPayload = {}
+      payload[await container.getNewAddress()] = '5@DFI'
+      payload[await container.getNewAddress()] = '5@DABC'
+      payload[await container.getNewAddress()] = '5@DDEF'
+
+      const data = await client.account.accountToAccount(from, payload)
+
+      expect(typeof data).toBe('string')
+      expect(data.length).toBe(64)
+    })
+
+    it('should accountToAccount with utxos', async () => {
+      const from = await container.getNewAddress()
+      await createToken(from, 'DGHI', 5)
```

Comment:
I was thinking to reuse DABC token. However, I faced this: **Error: RpcApiError: 'Execution test failed: not enough balance on owner's account, call utxostoaccount to increase it. ApplyAccountToAccountTx: amount 0.00000000 is less than 5.00000000', code: -32600** Using another token works! Probably I need to change the code in line 48

Reply:
Speaking of this, your test is not clean it is interdependent. 

Because you add tokens in between tests you are creating side-effect and populating the state/setup.
`'should getTokenBalances with pagination limit'` is populated now.

You should only create tokens during the setup process only. don't create new tokens in between test.